network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -377,26 +379,23 @@ class BanneredCampaignPage(PrimaryPage):
     use_intro_from_parent = models.BooleanField(
         default=False,
         blank=False,
-        help_text='This field will overwrite the intro field above with the intro field from this pages parent'
+        help_text='Overwrite the intro field above with the intro field from this pages parent'
     )
```

Comment:
What do you think about validating this field with a check to see if the pages parent does actually have an `intro` field? It might be better for the editors to have clear error message instead of surprising them with no intro showing up while it seems like they expect one, because they selected this option.

Reply:
Although, I think its better to be more explicit with whats happening so I've gone ahead an added that validation as you've suggested 😄